UR5MoveitJoints joints_input
geometry_msgs/Pose pick_pose
---
moveit_msgs/RobotTrajectory[] trajectories
